notification-svc/models/confirmationJob.ts

```diff
@@ -0,0 +1,56 @@
+import cron from 'node-cron'
+import { PrismaClient } from '@prisma/client';
+import sendEmail from './sendEmail'
+import { IAppointmentProps } from '../types/appointment.interface';
+
+const prisma = new PrismaClient();
+const confirmationJob = () => {
+    cron.schedule('0 12 * * *', async () => {
+
+        const today = new Date();
+        today.setHours(0, 0, 0, 0);
+
+        const twoDaysFromNow = new Date(today);
+        twoDaysFromNow.setDate(today.getDate() + 2);
+
+        const threeDaysFromNow = new Date(twoDaysFromNow);
+        threeDaysFromNow.setDate(twoDaysFromNow.getDate() + 1);
+
+        const appointmentsToConfirm = await prisma.appointment.findMany({
+            where: {
+                AND: [
+                    { status: "BOOKED" },
+                    {
+                        date: {
+                            gte: twoDaysFromNow,
+                            lt: threeDaysFromNow,
+                        },
+                    },
+                ]
+            },
+        });
```

Comment:
I think you should call our appointment API to fetch this data. Do you need an additional endpoint or can you use one already existing?

Reply:
Thats one way to do it, i am not sure if it would be a good idea especially if there are a lot of appointments to fetch.

Reply:
I asked Donovan, if we will allow all services to call DB directly, you should separate DB calls in DB-service to add an additional layer of protection.